docs/introduction/index.rst

```diff
@@ -41,6 +42,7 @@ With the Flocker API or CLI you can:
 * Migrate local data volumes between servers (currently Experimental)
 
 Flocker supports block-based shared storage such as Amazon EBS, Rackspace Cloud Block Storage, and EMC ScaleIO, as well as local storage (currently Experimental using our ZFS storage backend) so you can choose the storage backend that is best for your application.
+:ref:`View a complete list of supported storage backends <storage-backends>`.
```

Comment:
This link seems out-of-place here, in the manner of a "click here to see..." type link. I'd suggest linking this reference in the narrative text above it somewhere, e.g. "so you can choose any of the `supported storage backends <...>` that is best for your application..."